easybuild/easyconfigs/c/cram/cram-0.7-GCCcore-11.2.0.eb

```
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'cram'
+version = '0.7'
+
+homepage = 'https://bitheap.org/cram'
+description = "Cram is a functional testing framework for command line applications."
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['7da7445af2ce15b90aad5ec4792f857cef5786d71f14377e9eb994d8b8337f2f']
+
+builddependencies = [('binutils', '2.31.1')]
+
+multi_deps = {'Python': ['3.10.4', '2.7.18']}
+
+download_dep_fail = True
+use_pip = True
+
+fix_python_shebang_for = ['bin/cram']
+
+sanity_check_paths = {
+    'files': ['bin/cram'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = ["cram --help"]
+
+moduleclass = 'tools'
```

there should be a newline at the end of the file
```suggestion
moduleclass = 'tools'
```